bedrock/mozorg/templates/mozorg/about/forums/forums.html

```diff
@@ -73,13 +76,23 @@
         <li id="{{ forum.dashed }}">
           <div class="summary">
             <h4>{{ forum.id }}<a href="#{{ forum.dashed }}">#</a></h4>
-            <p class="description">{{ forum.description|safe }}</p>
+            <p class="description">
+              {%- if forum.id in mdn_forums -%}
```

Comment:
I wonder if it's worth revising what they are trying to do here, so it fits better with Gerv's system. This works, but trying to keep all the info that gets updated in one place seems like the way to go.